cinema/serializers.py

```diff
@@ -68,13 +78,64 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available"
         )
 
 
+class TicketCreateSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ticket
+        fields = ("row", "seat", "movie_session")
```

Comment:
`taken_places` should contain only row and seat (see the task), and movie session id - we already have it one level above (when getting one movie session)